pages/platform/ai/app_guide_02_getting_started/guide.en-asia.md

```diff
@@ -29,29 +29,29 @@ Click on the `Deploy an AI App`{.action} button and accept terms and conditions
 
 Once clicked, you will be redirected to the creation process detailed below.
 
-![AI App subscribe](images/ai_app_subscribe.png){.thumbnail}
+![AI Apps app subscribe](images/ai_app_subscribe.png){.thumbnail}
 
 #### Step 1: Select the location
 
 Select where your AI App will be hosted, meaning the physical location.
 
 > [!primary]
 >
-> OVHcloud provides multiple datacenters. You can find the capabilities for AI Apps in the guide [AI Apps capabilities](https://docs.ovh.com/asia/en/publiccloud/ai/apps/capabilities/).
+> OVHcloud provides multiple datacenters. You can find the capabilities for AI Apps in the guide [AI Apps capabilities](https://docs.ovh.com/asia/en/publiccloud/ai/deploy/capabilities/).
```

Comment:
This page does not exist yet. So we can't merge the PR as long as the related name changes and redirections have been done.